.travis/install.sh

```diff
@@ -27,8 +26,12 @@ then
     eval "$(pyenv init -)"
     if [ "${TOXENV}" = "py35" ]
     then
-        pyenv install 3.5.3  # Minimum version for OpenSSL 1.1.x
+        pyenv install 3.5.3
```

Comment:
please keep the comment as a reminder not to use anything less that 3.5.3 here.

e.g. 3.5.0 just won't compile with openssl 1.1.